### PR TITLE
Request Signature integration example

### PR DESCRIPTION
**Pull request by renatodelpupo**
The main goal is to add instructions and an example of integration with a server-side application for request signature purposes.

- [Request Signature integration](https://github.com/TrustlyInc/trustly-react-native-example/commit/3edbcfccba218c673f3c490e4938da1e6f1addeb)
- [Request Signature integration instructions](https://github.com/TrustlyInc/trustly-react-native-example/commit/96edacadf2687d29277b7ea9401233d96a1f2497)

Other contributions:

- [Environment variables setup](https://github.com/TrustlyInc/trustly-react-native-example/commit/72714c160765d5c87ba05c1142c7f88d032385da)
- [Establishment data type addition](https://github.com/TrustlyInc/trustly-react-native-example/commit/e37057353f1cbce23d109ab8d428b87aa8e0dae6)
- [Invalid amount value fix](https://github.com/TrustlyInc/trustly-react-native-example/commit/5e96e857fe85df362e7742c60585d1e628673591)
- [README instructions readability improvement](https://github.com/TrustlyInc/trustly-react-native-example/commit/9fc196e1a9d127a9889c6ec3304f6aa1f5b48fd8)

### CODE REVIEW

**Review by andresguedes — APPROVED** [review]
👍🏽

**Review by marcosrivereto — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+env.ts
```
Comment (lukevance) [inline .gitignore:1]: Love, ty! 😍

File: README.md [hunk 2]
```diff
@@ -101,6 +98,7 @@ The example below is from `RedirectActivity`
 ```
 
 ## How integrate native components with Trustly webview
+
 Some times you want to integrate the webview rendering Trustly widget with your native components in your application, like the image bellow:
```
Comment (lukevance) [inline README.md:102]: Let's change this to match public docs language:
> Trustly UI offers two types of user experiences that can be configured to fit your application. The primary method is to render the Select Bank Widget (shown below) which allows users to quickly search for and select a bank to begin the authorization process. See the [Trustly UI docs](https://amer.developers.trustly.com/payments/docs/sdk#select-bank-widget) for more details.

File: src/trustly.tsx [hunk 3]
```diff
@@ -26,26 +29,36 @@ export const widget = (accessId: String, data = {}) => `<!DOCTYPE html>
...
-    var TrustlyOptions = {
-      closeButton: false,
-      dragAndDrop: true,
-      widgetContainerId: "widget",
-    };
+export const lightbox = async (accessId: String, data: EstablishData) => {
+  if (!data) return;
 
-    Trustly.establish(establishData, TrustlyOptions);
+  // sign request - instructions available in readme.md
```
Comment (lukevance) [inline src/trustly.tsx:35]: Nice! 😎

**Review by lukevance — CHANGES_REQUESTED** [review]
Looks awesome! Just one suggestion for the documentation portion

File: README.md (hunk 2 above)
Comment (renatodelpupo) [reply README.md]: Done! 💪

**Review by lukevance — APPROVED** [review]
Sweet! 🍬 let's ship it 🚢

**Pull request merged**